libs/features/personalization/personalization.js

```diff
@@ -561,13 +561,25 @@ export const deleteMarkedEls = (rootEl = document) => {
     .forEach((el) => el.remove());
 };
 
+export function addSectionAnchors(rootEl = document) {
+  rootEl.querySelectorAll('.section-metadata').forEach((block) => {
+    [...block.children].forEach((row) => {
+      const col1 = row.children[0]?.textContent.toLowerCase().trim();
+      const col2 = row.children[1]?.textContent.toLowerCase().trim().replaceAll(' ', '-');
+      if (!col1 || !col2 || col1 !== 'anchor') return;
+      block.parentElement.setAttribute('id', col2);
```

Comment:
We'd end up running similar logic [here](https://github.com/adobecom/milo/blob/stage/libs/blocks/section-metadata/section-metadata.js#L60) if PZN is enabled, but at the same time I'm not sure it's worth adding logic in that file to check whether an ID already exists. So just a note for now

Reply:
section-metadata adds the `section-anchor` class and there's no perf impact so I think it's fine.